pkg/auth/azure.go

```diff
@@ -140,10 +140,13 @@ func StreamInterceptor(
 
 func HttpAuthMiddleWare(resp http.ResponseWriter, req *http.Request, jwks *keyfunc.JWKS, clientId string, tenantId string) error {
 	token := req.Header.Get("authorization")
-	err := ValidateToken(token, jwks, clientId, tenantId)
-	if err != nil {
-		resp.WriteHeader(http.StatusUnauthorized)
-		resp.Write([]byte("Invalid authorization header provided"))
+	if req.URL.Path != "/" {
```

Comment:
Why removing it from the root path, can you add more description please

Reply:
updated